packages/KLabel/KLabel.vue

```diff
@@ -1,12 +1,48 @@
 <template functional>
   <label class="k-input-label">
-    <slot />
+    <KoolTip
+      v-if="props.help"
+      :label="props.help"
+      class="label-tooltip"
+    >
+      <slot />
+      <KIcon
+        icon="help"
+        height="16"
+        width="16"
+        view-box="0 0 16 16"
+        hide-title />
+    </Kooltip>
+    <KoolTip
+      v-else-if="props.info"
+      :label="props.info"
+      class="label-tooltip"
+    >
+      <slot />
+      <KIcon
+        icon="info"
+        height="16"
+        width="16"
+        color="black"
```

Comment:
This is looking great @mheap , is there a reason why we are making this one explicitly black?

Reply:
@kaiarrowood Based on the intended context (KHCP) black is all that's needed. I'm happy to make this a prop with a default if you think it would be useful